plutus-use-cases/test/Spec/Vesting.hs

```diff
@@ -100,3 +262,8 @@ expectedError =
         maxPayment = Ada.lovelaceValueOf 20
         mustRemainLocked = Ada.lovelaceValueOf 40
     in InsufficientFundsError payment maxPayment mustRemainLocked
+
+
+-- Util
+delay :: Integer -> Trace.EmulatorTrace ()
+delay n = void $ Trace.waitNSlots $ fromIntegral n
```

Comment:
what do we do elsewhere? should this be in the library?

Reply:
I think a lot of code just calls `Trace.waitNSlots` and I suspect this was copied at some point from another contract that has a `WaitUntil` action. Relatedly, we've been debating making `WaitUntil` a native action that all contract models will have.

Reply:
Yeah, I was wondering that...